src/app/components/case/hearing-results/hearing-results.component.html

```diff
@@ -14,22 +14,25 @@
 
   <div *tab="'All Transcripts'; count: transcripts.length">
     <h2 class="govuk-heading-m">All transcripts for this case</h2>
-
-    <app-data-table id="transcriptsTable" [rows]="transcripts" [columns]="transcriptColumns">
-      <ng-template [tableRowTemplate]="transcripts" let-row>
-        <td class="govuk-table__cell">{{ row.hearingDate | date: 'dd MMM yyyy' }}</td>
-        <td class="govuk-table__cell">{{ row.type }}</td>
-        <td class="govuk-table__cell">{{ row.requestedOn | date: 'dd MMM yyyy HH:mm:ss' }}</td>
-        <td class="govuk-table__cell">{{ row.requestedBy }}</td>
-        <td class="govuk-table__cell">
-          <strong class="govuk-tag" [ngClass]="transcriptStatusClassMap[row.status]">
-            {{ row.status }}
-          </strong>
-        </td>
-        <td class="govuk-table__cell">
-          <a *ngIf="row.status === 'Complete'" [routerLink]="['transcripts', row.tra_id]">View</a>
-        </td>
-      </ng-template>
-    </app-data-table>
+    @if (transcripts.length) {
+      <app-data-table id="transcriptsTable" [rows]="transcripts" [columns]="transcriptColumns">
+        <ng-template [tableRowTemplate]="transcripts" let-row>
+          <td class="govuk-table__cell">{{ row.hearingDate | date: 'dd MMM yyyy' }}</td>
+          <td class="govuk-table__cell">{{ row.type }}</td>
+          <td class="govuk-table__cell">{{ row.requestedOn | date: 'dd MMM yyyy HH:mm:ss' }}</td>
+          <td class="govuk-table__cell">{{ row.requestedBy }}</td>
+          <td class="govuk-table__cell">
+            <strong class="govuk-tag" [ngClass]="transcriptStatusClassMap[row.status]">
+              {{ row.status }}
+            </strong>
+          </td>
+          <td class="govuk-table__cell">
+            <a *ngIf="row.status === 'Complete'" [routerLink]="['transcripts', row.tra_id]">View</a>
+          </td>
+        </ng-template>
+      </app-data-table>
+    } @else {
+      <p class="govuk-body">There are no transcripts for this case.</p>
+    }
```

Comment:
Not necessarily for now, but this could be part of the data-table component, checking `rows` input for length, with an overridable message for when it's empty. Thoughts?

Reply:
Yes, great suggestion Dave